src/components/MessageList/MessageList.js

```diff
@@ -285,11 +274,23 @@ class MessageList extends PureComponent {
                 .getMuteUserErrorNotification,
               getPinMessageErrorNotification: this.props
                 .getPinMessageErrorNotification,
+              members: this.props.members,
```

Comment:
curious why we're doing all of these individual namings instead of just `...this.props`

Reply:
yeah it ugly, someone gonna have a good time with this file lol